### PR TITLE
Update playground manifests

### PR DESCRIPTION
**Pull request by eerhardt**
Re-run eng\refreshManifests.ps1 with the latest changes to the playground apps

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/5744)

### CODE REVIEW

**Review by mitchdenny — APPROVED** [review]

File: playground/AWS/AWS.AppHost/aspire-manifest.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
       "stack-name": "AspireSampleDevResources",
       "template-path": "app-resources.template",
       "references": [
+        {
```
Comment (davidfowl) [inline playground/AWS/AWS.AppHost/aspire-manifest.json:9]: Looks like a bug

Comment (mitchdenny) [reply playground/AWS/AWS.AppHost/aspire-manifest.json:9]: Oh I didn't pick up that it was emitting twice!

Comment (davidfowl) [reply playground/AWS/AWS.AppHost/aspire-manifest.json:9]: cc @normj

Comment (normj) [reply playground/AWS/AWS.AppHost/aspire-manifest.json:9]: Here is a fix https://github.com/dotnet/aspire/pull/5748

**Pull request merged**